openedx/core/djangoapps/content/learning_sequences/api/tests/test_data.py

```diff
@@ -151,6 +160,55 @@ def test_empty_user_partition_groups(self):
             )
 
 
+class TestUsageKeyHashing(TestCase):
+    """
+    Basic sanity validation for usage key hashing functionality.
+    """
+    sequence_key = UsageKey.from_string('block-v1:A+B+C+type@sequential+block@D')
+    sequence_title = 'Sequence ABCD!'
+
+    def test_make_sequence_data_with_auto_hash(self):
+        """
+        If no usage_key_hash is specified, we calculate a default using hash_usage_key.
+        """
+        sequence_data = LearningSequenceData(
+            usage_key=self.sequence_key,
+            title=self.sequence_title,
+        )
+        assert sequence_data.usage_key == self.sequence_key
+        assert sequence_data.usage_key_hash == hash_usage_key(self.sequence_key)
```

Comment:
Nit, not required: You might want to actually calculate what the hash value should be separately and paste in in here. If for some reason there was an edge case for which `hash_usage_key` started returning `None`, this wouldn't catch it.

Reply:
Oh nvm, I see you do an explicit test right below.